File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1943.cpp
#include <iostream>

using namespace std;

int main(){
    int n;
    cin >> n;
    
    cout << "Top ";
    if(n <= 1) cout << 1 << endl;
    else if(n <= 3) cout << 3 << endl;
    else if(n <= 5) cout << 5 << endl;
    else if(n <= 10) cout << 10 << endl;
    else if(n <= 25) cout << 25 << endl;
    else if(n <= 50) cout << 50 << endl;
    else cout << 100 << endl;
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1943.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        int n = ler.nextInt();
        
        System.out.printf("Top ");
        if(n <= 1) System.out.printf("1\n");
        else if(n <= 3) System.out.printf("3\n");
        else if(n <= 5) System.out.printf("5\n");
        else if(n <= 10) System.out.printf("10\n");
        else if(n <= 25) System.out.printf("25\n");
        else if(n <= 50) System.out.printf("50\n");
        else System.out.printf("100\n");

    }
}